crates/ruff/src/rules/pycodestyle/snapshots/ruff__rules__pycodestyle__tests__E999_E999.py.snap

```diff
@@ -5,7 +5,8 @@ E999.py:3:1: E999 SyntaxError: unindent does not match any outer indentation lev
   |
 3 | def x():
 4 | 
-  |  E999
+  | ^ E999
```

Comment:
Is this intended?

Reply:
I can revert it and move it to another PR, but I thought having a caret rendered under the syntax error is nice. 

Why  I added it is because I had to introduce the `DisplaySyntaxError` struct to render `SyntaxErrors` with row/column information. I also changed `RustPython` to set the `location` for syntax errors to the start of the token causing the error (rather than past it???)